core/domain/wipeout_service.py

```diff
@@ -833,12 +953,12 @@ def _pseudonymize_suggestion_models(pending_deletion_request):
             request object to be saved in the datastore.
     """
     user_id = pending_deletion_request.user_id
-
     voiceover_application_class = (
         suggestion_models.GeneralVoiceoverApplicationModel)
+
     voiceover_application_models = voiceover_application_class.query(ndb.OR(
         voiceover_application_class.author_id == user_id,
-        voiceover_application_class.final_reviewer_id == user_id
+        voiceover_application_class.final_reviewer_id == user_id,
```

Comment:
Don't think you need the comma here? (Seems a bit weird to have it in a function call.)

Reply:
Done